lib/CL/pocl_util.h

```diff
@@ -159,6 +159,8 @@ cl_command_buffer_properties_khr
 pocl_cmdbuf_get_property (cl_command_buffer_khr command_buffer,
                           cl_command_buffer_properties_khr name);
 
+int pocl_is_cmdbuf_ready (cl_command_buffer_khr command_buffer);
```

Comment:
Seems the other cmdbuf helpers are named pocl_cmdbuf. While at it, can you extract those to pocl_cmdbuf.h to reduce pocl_util.h bloating? Also please add a comment to the new function.